interceptors/logging/interceptors.go

```diff
@@ -47,40 +50,102 @@ func (c *reporter) PostCall(err error, duration time.Duration) {
 	if err != nil {
 		logger = logger.With("error", fmt.Sprintf("%v", err))
 	}
-	logger.With(c.opts.durationFieldFunc(duration)...).Log(c.opts.levelFunc(code), fmt.Sprintf("finished %s %s call", c.kind, c.typ))
+
+	if !c.opts.shouldLogRequest(interceptors.FullMethod(c.service, c.method), err) {
+		logger.With(c.opts.durationFieldFunc(duration)...).Log(c.opts.levelFunc(code), fmt.Sprintf("finished %s %s call", c.kind, c.typ))
+	} else {
+		if c.isServer {
+			logger.With(c.opts.durationFieldFunc(duration)...).Log(c.opts.levelFunc(code), fmt.Sprintf("request finished - response handled %s %s", c.kind, c.typ))
+		} else {
+			logger.With(c.opts.durationFieldFunc(duration)...).Log(c.opts.levelFunc(code), fmt.Sprintf("response finished %s %s", c.kind, c.typ))
```

Comment:
do we really need `response` `request` specialization here? also what you mean response finished? How response can finish itself? (: read of response can be finished...

I would just do as before 
```
	logger.With(c.opts.durationFieldFunc(duration)...).Log(c.opts.levelFunc(code), fmt.Sprintf("finished %s %s call", c.kind, c.typ))
```

Reply:
What are the rationales for this/ (:

Reply:
* Firstly, I don't want to break the initial Logging of the middlewares, if we don't enable the request logging, we can make sure that the earlier logging works as it was working before this pr.

* For the recording the completion of request/response, I chose to curate the message for the above. However, I do agree that the messages can be modified 😅